lang/en/studentquiz.php

```diff
@@ -411,3 +428,70 @@
 $string['unapprove'] = 'Unapprove';
 $string['filter_label_question_creation_item'] = '{$a->creationtext} {$a->rowtext} {$a->inputtext}';
 $string['filter_label_question_creation_item_inputtext'] = '{$a->inputtext} {$a->inputtype}';
+$string['expandall'] = 'Expand all comments';
+$string['collapseall'] = 'Collapse all comments';
```

Comment:
> Language strings are not ordered in alphabetical order, that needs to be fixed.

I can't find the official dev documentation for it, but this is what I got a while back from https://tracker.moodle.org/browse/MDL-62822